fleet_vehicle_stock/views/fleet_vehicle.xml

```diff
@@ -5,9 +5,9 @@
     <record model="ir.ui.view" id="fleet_vehicle_form_view">
         <field name="name">fleet.vehicle.form (in fleet_vehicle_stock)</field>
         <field name="model">fleet.vehicle</field>
-        <field name="inherit_id" ref="fleet_vehicle_notebook.fleet_vehicle_form_view" />
+        <field name="inherit_id" ref="fleet.fleet_vehicle_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//notebook" position="inside">
+            <notebook>
```

Comment:
I think here we need to define the position inside

Reply:
https://github.com/odoo/odoo/blob/15.0/addons/fleet/views/fleet_vehicle_views.xml#L90